### ISSUE TITLE
Are sheriffs more likely to be central?

### ISSUE THREAD

**Issue by andymeneely** [andymeneely/chromium-history#239]
Let's look at developer centrality metrics (degree, closeness, and betweness) and see if those are correlated with number of sheriff hours. Do all of this at 2-month intervals - so sheriff hours within the time period and centrality based on a graph computed at the same 2-month interval.

It would also be interesting to see this time-shifted. For example, if I'm a sheriff in Jan-Feb, am I more likely to be central in Mar-Apr? 

Use the developer_adjacency table, and sheriff hours for this. 

Print the analysis in the nightly build

**Comment by kbaumzie** [comment]
We have found that the values for betweenness_centrality are quite smaller than we had anticipated. This is because of the way betweenness is calculated: it is measured in terms of a normalized percentage of the maximum possible betweenness a developer could have causing it's relativity to be much smaller. We just have to keep this in mind when we are analyzing the data for betweenness and it's relationship to sheriff hours.
